.editorconfig

```diff
@@ -0,0 +1,27 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+[*.py]
+max_line_length = 119
```

Comment:
The things you added in `[*]` are meant to be only for `[*.py]` right?
If so, better add them in `[*.py]`